### DISCUSSION TITLE
Macros not working

### DISCUSSION THREAD

**Discussion by XtremeSounds** [audacity/audacity#5505] [Q&A]
### Bug description

In 3.4 brand new today. macros do not output files nor does audacity create a default folder containing such nor recognize a user folder under preferences. Macros arent doing anything (Windows 11)

### Steps to reproduce

1. macro manager
2.  create new macro
3. add normalize
4. apply
5. click files and select the audio files you wish to process
6. nothing happens

### Expected behavior

a folder somewhere on my computer containing the new audio files with the desired effects 

### Actual behavior

nothing happens anywhere on my computer

### Audacity Version

current master / alpha / nightly build

### Operating system

Windows

**Comment by petersampsonaudacity** [comment]
Testing on W10 with 3.4.0

I cannot replicate this - WORKS-FOR-ME

You do realize that need to have an Export command in your Macro when you want a Macro that runs on files - if all you have is edit processing and no export then there will be no macro-output folder created.

![image](https://github.com/audacity/audacity/assets/50205984/b17a7e33-2587-4080-bce0-bd29c37ee375)


By default it should be created here:
![image](https://github.com/audacity/audacity/assets/50205984/6fce22d7-d79b-4a49-913f-fb5d066b17a0)


If you do not want the default location you specify a location of your choice in Directories Preferences:
https://manual.audacityteam.org/man/directories_preferences.html


_I tested both the default location and a custom location and both worked fine with the simple Macro shown above._

**Comment by XtremeSounds** [comment]
Ok thanks I didn’t know that

**Comment by XtremeSounds** [comment]
Quick question is there a good macro to improve the volume of samples. For some reason they always come out quiet. I am using normalize but have heard that this isnt always good

**Comment by petersampsonaudacity** [comment]
@XtremeSounds   you could try

a) https://manual.audacityteam.org/man/compressor.html

or

b) https://manual.audacityteam.org/man/loudness_normalization.html

Peter
